client/SDL/sdl_kbd.cpp

```diff
@@ -479,6 +470,26 @@ BOOL sdlInput::keyboard_grab(Uint32 windowID, SDL_bool enable)
 #endif
 }
 
+BOOL sdlInput::mouse_focus(Uint32 windowID)
+{
+	static Uint32 lastWindowID = -1;
```

Comment:
we should probably avoid that static variable and have a member for that ?

Reply:
good catch